Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/library/golang:1.19.4-alpine3.17 as builder
+WORKDIR /go/src/github.com/kiagnose/kubevirt-rt-checkup
+COPY . .
+RUN go build -v -o ./bin/kubevirt-rt-checkup ./cmd/
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.1.0-1656.1669627757
+
+RUN microdnf install -y shadow-utils && \
+    adduser --system --no-create-home -u 900 rt-checkup && \
+    microdnf remove -y shadow-utils && \
+    microdnf clean all
+
+COPY --from=builder /go/src/github.com/kiagnose/kubevirt-rt-checkup/bin/kubevirt-rt-checkup /usr/local/bin
```

Comment:
if the ENTRYPOINT is `kubevirt-rt-checkup` shouldn't you copy it to  `/kubevirt-rt-checkup`?
alternatively, if you're copying it to `/usr/local/bin`, why do you need an entrypoint?

Reply:
`/usr/local/bin` is in this image's `$PATH`, that is why the binary could simply be called by name and not using a full path like `/usr/local/bin/kubevirt-rt-checkup`.

The `ENTRYPOINT` stanza specifies the binary to execute (in this case `kubevirt-rt-checkup`).

Reply:
ah I thought it was the folder's name. my bad.